administrator/components/com_associations/views/associations/tmpl/modal.php

```diff
@@ -0,0 +1,157 @@
+			}
+		});
+	});"
+);
+?>
+<form action="<?php echo JRoute::_('index.php?option=com_associations&view=associations&layout=modal&tmpl=component&function=' . $function . '&' . JSession::getFormToken() . '=1');
+ ?>" method="post" name="adminForm" id="adminForm">
+
+<?php if (!empty( $this->sidebar)) : ?>
+	<div id="j-sidebar-container" class="span2">
+		<?php echo $this->sidebar; ?>
+	</div>
+	<div id="j-main-container" class="span10">
+<?php else : ?>
+	<div id="j-main-container">
+<?php endif;?>
+
+<?php echo JLayoutHelper::render('joomla.searchtools.default', array('view' => $this)); ?>
+	<?php if (empty($this->items)) : ?>
+		<div class="alert alert-no-items">
+			<?php echo JText::_('JGLOBAL_NO_MATCHING_RESULTS'); ?>
+		</div>
+	<?php else : ?>
+		<table class="table table-striped" id="associationsList">
+			<thead>
+				<tr>
+					<?php if (!is_null($this->component->fields->published)) : ?>
+						<th width="1%" class="center nowrap">
+							<?php echo JHtml::_('searchtools.sort', 'JSTATUS', 'published', $listDirn, $listOrder); $colSpan++; ?>
+						</th>
+					<?php endif; ?>
+					<th class="nowrap">
+						<?php echo JHtml::_('searchtools.sort', 'JGLOBAL_TITLE', 'title', $listDirn, $listOrder); ?>
+					</th>
+					<th width="15%" class="nowrap">
+						<?php echo JText::_('JGRID_HEADING_LANGUAGE'); ?>
+					</th>
+					<?php if (!is_null($this->component->fields->menutype)) : ?>
+						<th width="10%" class="nowrap">
+							<?php echo JHtml::_('searchtools.sort', 'COM_ASSOCIATIONS_HEADING_MENUTYPE', 'menutype_title', $listDirn, $listOrder); $colSpan++; ?>
+						</th>
+					<?php endif; ?>
+					<?php if (!is_null($this->component->fields->access)) : ?>
+						<th width="5%" class="nowrap hidden-phone">
+							<?php echo JHtml::_('searchtools.sort', 'JGRID_HEADING_ACCESS', 'access_level', $listDirn, $listOrder); $colSpan++; ?>
+						</th>
+					<?php endif; ?>
+					<th width="1%" class="nowrap hidden-phone">
+						<?php echo JHtml::_('searchtools.sort', 'JGRID_HEADING_ID', 'a.id', $listDirn, $listOrder); ?>
+					</th>
+				</tr>
+			</thead>
+			<tfoot>
+				<tr>
+					<td colspan="<?php echo $colSpan; ?>">
+						<?php echo $this->pagination->getListFooter(); ?>
+					</td>
+				</tr>
+			</tfoot>
+			<tbody>
+			<?php foreach ($this->items as $i => $item) :
+				?>
+				<tr class="row<?php echo $i % 2; ?>">
+					<?php if (!is_null($this->component->fields->published)) : ?>
+						<td class="center">
+							<span class="<?php echo $iconStates[$this->escape($item->published)]; ?>"></span>
+						</td>
+					<?php endif; ?>
+					<td class="nowrap has-context">
+						<?php if (isset($item->level)) : ?>
+							<?php echo JLayoutHelper::render('joomla.html.treeprefix', array('level' => $item->level)); ?>
+						<?php endif; ?>
+						<a class="select-link" href="javascript:void(0);" data-id="<?php echo $item->id; ?>">
+								<?php echo $this->escape($item->title); ?>
+						</a>
```

Comment:
because of extra space on mouse over, never add spaces or tabs before `</a>`
try to mouseover the link an you will see an extra space at the end of the link